interfaces/Portalicious/src/app/pages/projects-overview/project-summary-card/project-summary-card.component.html

```diff
@@ -0,0 +1,96 @@
+<p-card
+  styleClass="focus-within:border-primary hover:border-primary [&_.p-card-content]:flex-end group relative h-full border border-transparent *:h-full [&_.p-card-title]:flex-grow"
+>
+  <ng-template pTemplate="title">
+    <a
+      [routerLink]="projectLink(id())"
+      class="group-focus-within:text-primary group-hover:text-primary after:absolute after:inset-0 after:block"
+    >
+      @switch (project.status()) {
+        @case ('pending') {
+          <p-skeleton
+            class="my-1 cursor-progress text-xl"
+            height="1.25rem"
+            [width]="randomWidth"
+          />
+        }
+        @case ('success') {
+          <span
+            class="line-clamp-2 text-ellipsis"
+            [title]="project.data()?.titlePortal | translatableString"
+          >
+            {{ project.data()?.titlePortal | translatableString }}
+          </span>
+        }
+      }
+    </a>
+  </ng-template>
+  <ng-template pTemplate="content">
+    <dl class="flex flex-wrap divide-y border-b border-grey-300">
+      <div class="basis-1/2 border-t border-grey-300 px-1 py-2">
+        <app-project-metric-container
+          [status]="metrics.status()"
+          [value]="metrics.data()?.targetedPeople"
+          [label]="labels.targetedPeople"
```

Comment:
```suggestion
          label="Targeted people"
          i18n-label
```

I think this one slipped through the cracks (see my previous comment about removing `labels` from the component)